<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickerConfig</class>
 <widget class="QFrame" name="PickerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="aberrationEnduranceL">
     <property name="text">
      <string>色差忍耐值：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="aberrationEndurance"/>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="sceneAberrationThreshold">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="faceBestSizeL">
     <property name="text">
      <string>面部标准大小：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="faceBestSize">
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="faceMaxSizeL">
     <property name="text">
      <string>面部上限大小：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="faceMaxSize">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="faceExtendFactorL">
     <property name="text">
      <string>面部扩展系数：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="faceExtendFactor">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>3.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="faceSizeFactorL">
     <property name="text">
      <string>面部大小权重：</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="faceSizeFactor">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="faceShapeFactorL">
     <property name="text">
      <string>面部形状权重：</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="faceShapeFactor">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="faceAberrationFactorL">
     <property name="text">
      <string>面部色差系数：</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QDoubleSpinBox" name="faceAberrationFactor">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="faceReforceFactorL">
     <property name="text">
      <string>面部加强系数：</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QDoubleSpinBox" name="faceReforceFactor">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="faceLeftFactorL">
     <property name="text">
      <string>面部左方权重：</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="faceTopFactorL">
     <property name="text">
      <string>面部上方权重：</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="faceLeftFactor">
     <property name="minimum">
      <double>-10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QDoubleSpinBox" name="faceTopFactor">
     <property name="minimum">
      <double>-10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="faceMinSizeL">
     <property name="text">
      <string>面部下限大小：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="faceMinSize">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sceneAberrationThresholdL">
     <property name="text">
      <string>场景色差阈值：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="actionAberrationThresholdL">
     <property name="text">
      <string>动作色差阈值：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="actionAberrationThreshold">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
